public/js/cube.js

```diff
@@ -1271,7 +1271,11 @@ Cube.prototype.dataCache = function() {
 };
 
 Cube.prototype.clearDataCache = function() {
-	this.baseModel().data = {};
```

Comment:
All data requests should be caching to the baseModel. This is very important as it is what ensures that you will get the same results when you open that model again after a save.